### PR TITLE
[UDA] Note that action signature is mandatory

### PR DESCRIPTION
**Pull request by charislam**
# Description

You must include an action signature for your job, even if you don't need to use the `job_id` and `config` arguments.

Also removed some repetition that is hard to maintain. (The UDA examples have already been moved into how-to guides.) Left the UDA page in concepts for now, but these will also be migrated over time.

Also some drive-by formatting changes to test the automated formatting set-up.

# Links

Fixes #[insert issue link, if any]

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Are procedure and highlight tags used appropriately?
- [ ] Has the index been updated appropriately?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] Are all links provided in reference style, and resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: timescaledb/how-to-guides/user-defined-actions/create-and-register.md [hunk 1]
```diff
@@ -1,44 +1,54 @@
...
 execute. Then wrap your commands in a `CREATE OR REPLACE` statement. You can
 also define the language of your commands in this statement. The example below
 uses PLPGSQL.
 
 The outer statement contains the action signature: `(job_id INT, config JSONB)`.
-Include the signature literally. Don't replace with an actual job ID or config
-object. You define `config` when you register your job.  `job_id` is also
-automatically generated then.
+You must include this signature even if you don't need to use those arguments in
+your action. Include it literally. Don't replace with an actual job ID or config
```
Comment (Loquacity) [inline timescaledb/how-to-guides/user-defined-actions/create-and-register.md:18]:
```suggestion
your action. Include it literally, don't replace it with an actual job ID or config
```

File: timescaledb/how-to-guides/user-defined-actions/create-and-register.md [hunk 2]
```diff
@@ -1,44 +1,54 @@
...
+object. The config and job ID are defined and automatically generated later.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/user-defined-actions/create-and-register.md:19]:
```suggestion
object. The config and job ID are automatically defined and generated later.
```

??

File: timescaledb/how-to-guides/user-defined-actions/create-and-register.md [hunk 3]
```diff
@@ -1,44 +1,54 @@
...
 
 The following example defines a simple procedure that raises a notice. Replace
```
Comment (Loquacity) [inline timescaledb/how-to-guides/user-defined-actions/create-and-register.md:21]:
```suggestion
This example defines a simple procedure that raises a notice. Replace
```

File: timescaledb/how-to-guides/user-defined-actions/create-and-register.md [hunk 4]
```diff
@@ -1,44 +1,54 @@
...
 the `RAISE NOTICE` with the commands you want to run.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/user-defined-actions/create-and-register.md:22]:
```suggestion
the `RAISE NOTICE` command with the commands you want to run.
```

File: timescaledb/how-to-guides/user-defined-actions/create-and-register.md [hunk 5]
```diff
@@ -1,44 +1,54 @@
...
 
 <procedure>
 
 ## Defining a procedure
+
 1.  Write the commands you want your job to run.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/user-defined-actions/create-and-register.md:28]:
```suggestion
1.  Write the commands you want your job to run. For example:
```

File: timescaledb/how-to-guides/user-defined-actions/create-and-register.md [hunk 6]
```diff
@@ -1,44 +1,54 @@
...
+
     ```sql
     BEGIN
         RAISE NOTICE 'Executing job % with config %', job_id, config;
     END
     ```
-2.  Wrap the commands in a `CREATE OR REPLACE PROCEDURE` statement. 
+
+1.  Wrap the commands in a `CREATE OR REPLACE PROCEDURE` statement.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/user-defined-actions/create-and-register.md:36]:
```suggestion
1.  Wrap the commands in a `CREATE OR REPLACE PROCEDURE` statement. For example:
```

File: timescaledb/how-to-guides/user-defined-actions/create-and-register.md [hunk 7]
```diff
@@ -47,10 +57,13 @@ parameters, use a NULL config.
 <procedure>
 
 ## Registering an action
+
 1.  Add a job for your user-defined action.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/user-defined-actions/create-and-register.md:61]:
```suggestion
1.  Add a job for your user-defined action. For example:
```

**Review by Loquacity — APPROVED** [review]

**Review by davidkohn88 — COMMENTED** [review]
This looks fine but I have some general feedback around the structure and wording of these docs, would you like it here?

**Comment by Loquacity** [comment]
> This looks fine but I have some general feedback around the structure and wording of these docs, would you like it here?

Yes, absolutely! 😁

**Comment by davidkohn88** [comment]
Rethinking this, let’s get the small fix in and I can open a new PR around
this.

**Review by davidkohn88 — APPROVED** [review]

**Pull request merged**